Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/lists/modular/mods/SavedStateMod.kt

```diff
@@ -15,26 +15,28 @@ class SavedStateMod<T : ModularAdapter.VH> :
 
     private val savedStates = mutableMapOf<String, Parcelable>()
 
+    private var initial = true
+
     override fun onPostBind(adapter: ModularAdapter<T>, vh: T, pos: Int) {
```

Comment:
if I got it right, the change of SavedStateMod does not only affect the Statistic Cards but all Lists in the App...
Under normal circumstances.
as this Mod is currently only used by the statistics, maybe you could add a comment or something

Reply:
@chiljamgossow 
I will approve it, but please add a comment in some PR later